### PR TITLE
Added the Tech Outpost

### PR DESCRIPTION
**Pull request by Mailaender**
without the defensive turret for now.

### CODE REVIEW

File: rules/tech-structures.yaml [hunk 1]
```diff
@@ -40,3 +40,49 @@ cahosp:
...
+#	WithRangeCircle:
+#		Range: 12c0
+#		Type: patriot
+#	AutoTarget:
+#	Turreted:
+#		TurnSpeed: 10
+#		InitialFacing: 224
+#		Offset: 550,0,0
+#	AttackTurreted:
+#		Voice: Attack
```
Comment (abcdefg30) [inline rules/tech-structures.yaml]: Um, this actor is not voiced I think.

**Comment by abcdefg30** [comment]
So, what's the problem with repairing and having a turret at the same time?
(I didn't spot something obviously wrong during a quick test.)

**Comment by Mailaender** [comment]
I don't remember. It has been almost a month now. Could be targeting and the rallypoint conflicting. Enabled it nonetheless as it doesn't crash the game or something otherwise serious.

File: rules/tech-structures.yaml (hunk 1 above)
Comment (Mailaender) [inline rules/tech-structures.yaml]: Removed it.

**Pull request closed without merging**